packages/manager/apps/web/client/app/hosting/database/private-sql-activation/private-sql-activation.controller.js

```diff
@@ -0,0 +1,128 @@
+import filter from 'lodash/filter';
+import find from 'lodash/find';
+import map from 'lodash/map';
+
+export default class PrivateSqlActivationController {
+  /* @ngInject */
+  constructor($q, $translate, OrderService, OvhApiHostingWeb) {
+    this.$q = $q;
+    this.$translate = $translate;
+    this.OrderService = OrderService;
+    this.OvhApiHostingWeb = OvhApiHostingWeb;
+  }
+
+  $onInit() {
+    this.loading = {
+      hosting: false,
+      checkout: false,
+    };
+    this.error = {
+      hosting: null,
+      checkout: null,
+    };
+
+    if (!this.availability) {
```

Comment:
Could be done in routing with `redirectTo` (cf https://github.com/ovh/manager/blob/master/packages/manager/modules/pci/src/projects/project/streams/onboarding/onboarding.routing.js#L7-L15)